compiler-plugin/src/main/scala/scalapb/compiler/DescriptorImplicits.scala

```diff
@@ -595,16 +596,20 @@ class DescriptorImplicits private[compiler] (
     def sealedOneofTypeMapper =
       sealedOneofTraitScalaType / (sealedOneofTraitScalaType.name + "TypeMapper")
 
-    private[this] val valueClassNames = Set("AnyVal", "scala.AnyVal", "_root_.scala.AnyVal")
+    private[this] val valueClassNames     = Set("AnyVal", "scala.AnyVal", "_root_.scala.AnyVal")
```

Comment:
Please move to `object DescriptorImplicits` below - there are a bunch of other constants there; first letter of the name Capitalized.